pkg/controller/machine/controller.go

```diff
@@ -132,7 +132,7 @@ func (c *MachineControllerImpl) create(machine *clusterv1.Machine) error {
 		return err
 	}
 
-	return c.actuator.Create(cluster, machine)
+	return c.actuator.Create(cluster, machine, nil)
```

Comment:
We need a solution where the machine controller can create with specified cert if we want this to work for clusterctl bootstrapping.

Reply:
Yes, I will create an Issue for this working out of gce-machine-controller. We need that for master replacement and our eventual support of multi-master as well.

Reply:
Let's take a step back. For what issue are you making these changes? If it is for bringing things to alpha, we should think about whether alpha means gcp-deployer functionailty of clusterctl functionality. If the latter, this change is not really addressing the issue.

@rsdcastro - Not sure if you have thoughts on this

Reply:
The issue is https://github.com/kubernetes-sigs/cluster-api/issues/52. This PR is a step along the way to alpha in that it brings in the desired functionality for one cloud in an undesired location (gcp-deployer). I'll create a new issue for implementing these changes inside the clusterctl once it is functional and ready for multiple developers to work with.